docs/docs/train-tutorial.md

```diff
@@ -0,0 +1,338 @@
+    ```
+* `events.out.tfevents.1632250686.ip-172-31-15-179.14229.0`: [TensorBoard](https://www.tensorflow.org/tensorboard/get_started) logs
+
+## Step-by-step tutorial
+
+### 1. Specify the path to your videos 
+
+Save all of your videos in one folder.
+
+* Your videos should all be saved in formats that are suppored by FFMPEG, [which are listed here](https://www.ffmpeg.org/general.html#Supported-File-Formats_002c-Codecs-or-Features).
+* Your video folder must contain only valid video files, since zamba will try to load all of the files in the directory.
+* Your videos must all be in the top level of the video folder - `zamba` does not extract videos from nested directories.
+
+Add the path to your video folder with `--data-dir`. For example, if your videos are in a folder called `example_vids`, add `--data-dir example_vids/` to your command.
+
+In Python, the data directory is specified when `TrainConfig` is instantiated:
+
+```python
+from zamba.models.config import TrainConfig
+
+# note this will not run yet because labels are not specified
+train_config = TrainConfig(data_directory='example_vids/')
+```
+
+### 2. Specify your labels
+
+Your labels should be saved in a `.csv` file with columns for filepath and label. For example:
+
+```console
+$ cat example_labels.csv
+filepath,label
+example_vids/eleph.MP4,elephant
+example_vids/leopard.MP4,leopard
+example_vids/blank.MP4,blank
+example_vids/chimp.MP4,chimpanzee_bonobo
+```
+
+Add the path to your labels with `--labels`.  For example, if your videos are in a folder called `example_vids` and your labels are saved in `example_labels.csv`:
+
+```console
+$ zamba train --data-dir example_vids/ --labels example_labels.csv
+```
+
+In Python, the labels are passed in when `TrainConfig` is instantiated. The Python package allows you to pass in labels as either a file path or a pandas dataframe:
+```python
+labels_dataframe = pd.read_csv('example_labels.csv').set_index('filepath')
+train_config = TrainConfig(data_directory='example_vids/', labels=labels_dataframe)
+```
+
+#### Labels `zamba` has seen before
+
+Your labels may be included in the list of [`zamba` class labels](models.md#species-classes) that the provided models are trained to predict. If so, the relevant model that ships with `zamba` will essentially be used as a checkpoint, and model training will resume from that checkpoint.
+
+#### Completely new labels
+
+You can also train a model to predict completely new labels - the world is your oyster! (We'd love to see a model trained to predict oysters.) If this is the case, the model architecture will replace the final [neural network](https://www.youtube.com/watch?v=aircAruvnKk&t=995s) layer with a new head that predicts *your* labels instead of those that ship with `zamba`. [Backpropogation](https://www.youtube.com/watch?v=Ilg3gGewQ5U) will continue from that point with the new head.
```

Comment:
:laughing:

Reply:
we might want to provide additional details around the finetuning use case or provide a link to read more, eg. https://keras.io/guides/transfer_learning/